packages/maskbook/src/plugins/GoodGhosting/UI/GameStatsView.tsx

```diff
@@ -0,0 +1,174 @@
+import { formatBalance } from '@masknet/web3-shared'
+import { makeStyles, Grid, Typography, Box, Button } from '@material-ui/core'
+import { addSeconds, differenceInDays, formatDuration } from 'date-fns/esm'
```

Comment:
plz split to multi import statement

CC @Jack-Works

Reply:
Split the date-fns imports to individual statements :+1: